csharp/Platform.Collections/Arrays/ArrayPool.cs

```diff
@@ -2,6 +2,10 @@
 
 namespace Platform.Collections.Arrays
 {
+    /// <summary>
+    /// <para>Represents a set of wrapper methods over class methods <see cref="ArrayPool{T}"> to simplify access to them.</para>
```

Comment:
`<see cref="ArrayPool{T}"/> class methods` instead of `class methods <see cref="ArrayPool{T}">`